src/txmempool.cpp

```diff
@@ -1557,16 +1524,18 @@ bool CCoinsViewMemPool::GetSidechain(const uint256& scId, CSidechain& info) cons
                 //info.creationBlockHash doesn't exist here!
                 info.creationBlockHeight = -1; //default null value for creationBlockHeight
                 info.creationTxHash = scCreationHash;
-                info.creationData.withdrawalEpochLength = scCreation.withdrawalEpochLength;
-                info.creationData.customData = scCreation.customData;
-                info.creationData.constant = scCreation.constant;
-                info.creationData.wCertVk = scCreation.wCertVk;
-                info.creationData.wCeasedVk = scCreation.wCeasedVk;
-                info.creationData.vFieldElementCertificateFieldConfig = scCreation.vFieldElementCertificateFieldConfig;
-                info.creationData.vBitVectorCertificateFieldConfig = scCreation.vBitVectorCertificateFieldConfig;
-                info.creationData.forwardTransferScFee = scCreation.forwardTransferScFee;
-                info.creationData.mainchainBackwardTransferRequestScFee = scCreation.mainchainBackwardTransferRequestScFee;
-                info.mainchainBackwardTransferRequestDataLength = scCreation.mainchainBackwardTransferRequestDataLength;
+                info.fixedParams.withdrawalEpochLength = scCreation.withdrawalEpochLength;
+                info.fixedParams.customData = scCreation.customData;
+                info.fixedParams.constant = scCreation.constant;
+                info.fixedParams.wCertVk = scCreation.wCertVk;
+                info.fixedParams.wCeasedVk = scCreation.wCeasedVk;
+                info.fixedParams.vFieldElementCertificateFieldConfig = scCreation.vFieldElementCertificateFieldConfig;
+                info.fixedParams.vBitVectorCertificateFieldConfig = scCreation.vBitVectorCertificateFieldConfig;
+                info.pastEpochTopQualityCertView.forwardTransferScFee = CScCertificate::INT_NULL;
+                info.pastEpochTopQualityCertView.mainchainBackwardTransferRequestScFee = CScCertificate::INT_NULL;
```

Comment:
Do we need to initialize `pastEpochTopQualityCertView` fields with `INT_NULL`? seems the default constructor will do it.

Reply:
Fixed.